Allow to start a sprint at a certain time in the next hour

moritzuehling opened the pull request:
This should allow to add support to have a sprint start at an explicit time.

In our discord, we tend to ask for a sprint by asking something along the lines of

> Anyone wanna sprint at :30

I want to support that use-case with the bot, too. So in addition to the `sprint for 20 in 3` syntax, I've added `sprint for 20 at :30` (or `sprint for 20 at 30`) as a possible syntax for the command. The bot then does the necessary calculations to figure out the start-time in minutes, and uses existing code-paths.

cwarwicker commented:
Hi, I'm happy to include this, but it will suffer from the fact that the timing will be done on the server. Most timezones have the same minute value, but some don't. E.g: https://www.timeanddate.com/worldclock/

I doubt it will be a big issue, but it's something to be aware of.

Your other option is getting the user to set their timezone first and calculating it properly for them, like it does in the event command, but that may be overkill for this.

moritzuehling commented:
I've added a `(UTC)` in the help strings. Looking at the list, I had forgotten about our Indian and Australian users.

I wrongly remembered "Oh, it's just North Korea and some super small countries," but you can't really say this about India, and not even the North Korea bit was correct - they've since adapted standard time again. Whoops!

As a small stopgap, I added `(UTC)` at the end of the example, to make it clear that it'll use that as basis. It should still work for most users, and the remaining ones still get told how many minutes the bot calculated in the response message:

![image](https://user-images.githubusercontent.com/7847742/61977428-bb3e2480-afee-11e9-95e6-78623996d5f5.png)
